modules/core/test/v2/unit/coins/ltc.ts

```diff
@@ -63,21 +63,24 @@ describe('LTC:', function() {
       version2Address.should.equal(bech32Address);
     });
 
-    it('upper case bech32 mainnet address', function() {
-      // bech32 addresses which are all upper case or all lower case are potentially valid,
-      // but mixed case is always invalid. Canonical bech32 addresses should be all lower case however
-      const bech32Address = 'LTC1QGRL8ZPNDSKLAA9SWGD5VEVYXMX5X63VCRL7DK4';
-      const newAddress = ltc.canonicalAddress(bech32Address);
-      newAddress.should.equal(bech32Address.toLowerCase());
+    it('non-lowercase bech32 mainnet address should fail', function () {
+      // bech32 addresses which are non-lowercase are invalid,
```

Comment:
Might make sense to put some detail about how we only accept "canonical" lower case bech32 addresses, even though they are "valid" according to the spec, akin to the old comment. I know that if I came across this line without any context I'd be confused since I know upper case bech32 addresses to be valid according to the spec.

Reply:
@sangaman good idea. test description has been updated